### PR TITLE
README overhaul

### PR DESCRIPTION
**Pull request by tapir**

### CODE REVIEW

**Comment by tapir** [comment]
Any comments?

File: README.md [hunk 1]
```diff
@@ -1,30 +1,27 @@
...
 
 import (
 	"runtime"
-
 	glfw "github.com/go-gl/glfw3"
 )
 
 func init() {
+	// This is needed to arrange that main() runs on main thread.
+	// See documentation for functions that are only allowed to be called from the main thread.
```
Comment (dmitshur) [inline README.md:24]: This is a better place to mention that, :+1:.

File: README.md [hunk 2]
```diff
@@ -50,38 +47,78 @@ func main() {
...
+* Added function `Window.SetCursor`.
+* Added function `Window.GetFrameSize`.
+* Added window hint `Floating`.
+* Added window hint `AutoIconify`.
+* Added window hint `ContextReleaseBehavior`.
+* Added window hint `DoubleBuffer`.
+* Added hint value `AnyReleaseBehavior`.
+* Added hint value `ReleaseBehaviorFlush`.
+* Added hint value `ReleaseBehaviorNone`.
+* Added hint value `DontCare`.
```
Comment (dmitshur) [inline README.md:72]: It's much more readable as 2 sections. :+1:

**Comment by dmitshur** [comment]
LGTM!

**Pull request merged**